rero_ils/modules/patrons/api.py

```diff
@@ -191,11 +191,29 @@ def update(self, data, dbcommit=False, reindex=False):
         data = trim_barcode_for_record(data=data)
         # synchronize the rero id user profile data
         data = dict(self, **data)
+        old_keep_history = self.keep_history
         self.sync_user_and_profile(data)
         super(Patron, self).update(data, dbcommit, reindex)
         self._update_roles()
+        self. _anonymize_loans(
```

Comment:
is there a space between '.' and '_' ?
As `_anonymize_loans` is a class method, it's better/required to call it with `Patron._anonymize_loans`(not self)